website/public/locales/de/common.json

```diff
@@ -1,36 +1,61 @@
 {
   "about": "Über",
   "account_settings": "Konto",
-  "admin_dashboard": "Admin-Bereich",
+  "admin_dashboard": "Admin-Dashboard",
+  "back_to_dashboard": "Zum Dashboard zurückkehren",
+  "bug_description": "Wenn Sie Daten beitragen wollten aber hier gelandet bist, melde bitte einen Fehler.",
+  "bug_button": "Fehler Melden",
```

Comment:
"melden" lowercase

Reply:
That button uses Title case. In English it's "Report a Bug", so I'd think German should also capitalize the verbs. (Or do we not have title case in German?)